packages/wrangler/src/publish/publish.ts

```diff
@@ -6,14 +6,15 @@ import chalk from "chalk";
 import tmp from "tmp-promise";
 import { bundleWorker } from "../bundle";
 import { printBundleSize } from "../bundle-reporter";
-import { fetchListResult, fetchResult } from "../cfetch";
+import { type FetchError, fetchListResult, fetchResult } from "../cfetch";
```

Comment:
To stay inline with our conventions the pattern for importing types is using the `import type` 
```suggestion
import { fetchListResult, fetchResult } from "../cfetch";
import type { FetchError } from "../cfetch";
```

Reply:
Is there something wrong with the `import { type ..., ... }` pattern? I think I've used that in various places

Reply:
Fixed all these

Reply:
> Is there something wrong with the `import { type ..., ... }` pattern? I think I've used that in various places

@penalosa consistency, we started with the `import type { FetchError } from "../cfetch";` and would like to keep it consistent, we should at it as a ESLint rule and have Prettier handle it if it can.